plugins/javatemplates/src/main/java/org/apache/struts2/views/java/simple/ScriptHandler.java

```diff
@@ -0,0 +1,53 @@
+package org.apache.struts2.views.java.simple;
+
+import com.opensymphony.xwork2.ActionContext;
+import com.opensymphony.xwork2.util.ValueStack;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.struts2.interceptor.csp.DefaultCspSettings;
+import org.apache.struts2.views.java.Attributes;
+import org.apache.struts2.views.java.TagGenerator;
+
+import java.io.IOException;
+import java.util.Map;
+
+public class ScriptHandler extends AbstractTagHandler implements TagGenerator {
+
+    @Override
+    public void generate() throws IOException {
+        Map<String, Object> params = context.getParameters();
+        Attributes attrs = new Attributes();
+
+        // TODO check that
+        // Order of execution of the interceptor
```

Comment:
I think we can remove this todo - since we verified it with our tests and the sample vulnerable application!

Reply:
okok